### PR TITLE
Update the workloads API versions to 1.9 for rest of the docs

### PR DESCRIPTION
**Pull request by cimomo**
Update the workloads API versions to 1.9 for rest of the docs.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7099)

### CODE REVIEW

File: docs/tasks/inject-data-application/podpreset-replicaset.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta2
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
```
Comment (tengqm) [inline docs/tasks/inject-data-application/podpreset-replicaset.yaml]: please consider remove the comment because it is confusing and inaccurate

Comment (cimomo) [reply docs/tasks/inject-data-application/podpreset-replicaset.yaml]: Sure, removed the comment. @tengqm

**Comment by cimomo** [comment]
@tengqm Could you take a look?

File: docs/admin/authentication.md [hunk 2]
```diff
@@ -185,7 +185,7 @@ talk to the API server. Accounts may be explicitly associated with pods using th
 NOTE: `serviceAccountName` is usually omitted because this is done automatically.
 
 ```
-apiVersion: apps/v1beta2
+apiVersion: apps/v1
```
Comment (heckj) [inline docs/admin/authentication.md]: Looks good - can we add a comment that this is specific to version 1.9? Perhaps something like:

> `apiVersion: apps/v1 # this apiVersion is relevant as of Kubernetes 1.9`

Comment (cimomo) [reply docs/admin/authentication.md]: Sounds good. Will update.

**Comment by cimomo** [comment]
@heckj Please take another look.

**Pull request merged**